### DISCUSSION TITLE
In-app config switching

### DISCUSSION THREAD

**Discussion by QuentinMcCarthy** [ThioJoe/YT-Spammer-Purge#1030] [Ideas]
Currently the only way to switch config files is to close the app entirely and restart it, which can be a bit annoying especially as the app always resets its position on-screen.

It'd be useful if there was a way to return to the config switching menu from any prompt, in the same way you can return to the main menu from any prompt. This'd allow quick and easy config switching, helpful for switching between say, a self-channel/video scan config, to a general scan config, to the default config.

**Comment by ThioJoe** [comment]
It would be really difficult to do this based on the structure of the program, so this probably won't happen
